src/settings-ui/Settings.UI/SettingsXAML/Views/ZoomItPage.xaml

```diff
@@ -0,0 +1,218 @@
+﻿<Page
+    x:Class="Microsoft.PowerToys.Settings.UI.Views.ZoomItPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:controls="using:Microsoft.PowerToys.Settings.UI.Controls"
+    xmlns:converters="using:Microsoft.PowerToys.Settings.UI.Converters"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:tkcontrols="using:CommunityToolkit.WinUI.Controls"
+    xmlns:ui="using:CommunityToolkit.WinUI"
+    AutomationProperties.LandmarkType="Main"
+    mc:Ignorable="d">
+
+    <Page.Resources>
+        <converters:ZoomItInitialZoomConverter x:Key="ZoomItInitialZoomConverter" />
+        <converters:ZoomItTypeSpeedSliderConverter x:Key="ZoomItTypeSpeedSliderConverter" />
+    </Page.Resources>
+
+    <controls:SettingsPageControl
+        x:Uid="ZoomIt"
+        IsTabStop="False"
+        ModuleImageSource="ms-appx:///Assets/Settings/Modules/ZommIt.png">
+        <controls:SettingsPageControl.ModuleContent>
+            <StackPanel ChildrenTransitions="{StaticResource SettingsCardsAnimations}" Orientation="Vertical">
+                <tkcontrols:SettingsCard
+                    x:Uid="ZoomIt_EnableToggleControl_HeaderText"
+                    HeaderIcon="{ui:BitmapIcon Source=/Assets/Settings/Icons/ZoomIt.png}"
+                    IsEnabled="{x:Bind ViewModel.IsEnabledGpoConfigured, Mode=OneWay, Converter={StaticResource BoolNegationConverter}}">
+                    <ToggleSwitch x:Uid="ToggleSwitch" IsOn="{x:Bind ViewModel.IsEnabled, Mode=TwoWay}" />
+                </tkcontrols:SettingsCard>
+                <InfoBar
+                    x:Uid="GPO_SettingIsManaged"
+                    IsClosable="False"
+                    IsOpen="{x:Bind ViewModel.IsEnabledGpoConfigured, Mode=OneWay}"
+                    IsTabStop="{x:Bind ViewModel.IsEnabledGpoConfigured, Mode=OneWay}"
+                    Severity="Informational" />
+                <InfoBar
+                    x:Uid="ZoomIt_SimultaneousStandaloneZoomItWarning"
```

Comment:
Can we show this only if standalone version is really running? The we can severity warning.

Reply:
The idea is that by seeing this users might remember to not try to start sysinternals tools after starting this one. So we'd want it to be always visible